### PR TITLE
Add documentaion related to sso templates 

### PR DESCRIPTION
**Pull request by AnjanaSamindraPerera**

### CODE REVIEW

File: en/includes/guides/authentication/index.md [hunk 1]
```diff
@@ -41,4 +43,8 @@ With identity federation, users can use an existing user account registered to a
 
 You can configure any number of external connections for your application via {{ product_name }}.
 
-{% include "../fragments/manage-connection/add-groups.md" %}
\ No newline at end of file
+{% include "../fragments/manage-connection/add-groups.md" %}
+
+
+## hjgjhghj
```
Comment (NipuniBhagya) [inline en/includes/guides/authentication/index.md:49]: Let's remove the unnecessary title and the lines here.

**Review by himeshsiriwardana — APPROVED** [review]

File: en/includes/guides/authentication/index.md (hunk 1 above)
Comment (himeshsiriwardana) [inline en/includes/guides/authentication/index.md:49]:
```suggestion
```

**Review by himeshsiriwardana — APPROVED** [review]

File: en/includes/guides/authentication/sso-integrations/add-google-workspace-template.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Log in to Google Workspace using the {{ product_name }}
+
+This page guides you through using {{ product_name }} to log in to Google.
```
Comment (NipuniBhagya) [inline en/includes/guides/authentication/sso-integrations/add-google-workspace-template.md:3]:
```suggestion
This page provides step-by-step instructions on how to use {{ product_name }} for logging in to Google.
```

File: en/includes/guides/authentication/sso-integrations/add-google-workspace-template.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+!!! note
+    You need to have a Google domain. Further, ensure you have a Google Workspace account and your account supports SSO by referring to the [Google documentation](https://support.google.com/a/answer/182076?sjid=13406338490513240799-AP){:target="_blank"}.
```
Comment (NipuniBhagya) [inline en/includes/guides/authentication/sso-integrations/add-google-workspace-template.md:6]:
```suggestion
    To get started, you need to have a Google domain. Further, ensure you have a Google Workspace account and your account supports SSO by referring to the [Google documentation](https://support.google.com/a/answer/182076?sjid=13406338490513240799-AP){:target="_blank"}.
```

**Pull request merged**